src/orderbook.js

```diff
@@ -1197,6 +1336,44 @@ OrderBook.prototype.computeAutobridgedOffers = function() {
   this._offersAutobridged = autobridgeCalculator.calculate();
 };
 
+OrderBook.prototype.computeAutobridgedOffersWrapper = function() {
+  var startTime = Date.now();
+  this.computeAutobridgedOffers();
+  this.mergeDirectAndAutobridgedBooks();
+  var lasted = (Date.now() - startTime);
+  
+  const newMult =
+    ((lasted * 2 / OrderBook.AUTOBRIDGE_CALCULATE_THROTTLE_TIME) << 0) + 1;
+  if (newMult !== this._autobridgeThrottleTimeMultiplier) {
+    this._autobridgeThrottleTimeMultiplier = newMult;
+    this.createDebouncedOffersWrapper();
+  }
+}
+
+OrderBook.prototype.createDebouncedOffersWrapper = function() {
+  const m = this._autobridgeThrottleTimeMultiplier;
+  this.computeAutobridgedOffersThrottled = 
+    _.debounce(
```

Comment:
If there is way to know what transaction is last in batch then yes, we don't need that debounce thing.
But we still need throttle, to save user's device from burning in case it's slow.